x-pack/plugins/remote_clusters/public/sections/remote_cluster_list/detail_panel/detail_panel.js

```diff
@@ -34,25 +40,63 @@ export class DetailPanelUi extends Component {
     super(props);
   }
 
-  renderTabs() {
+  renderCluster() {
     const { cluster } = this.props;
+    const { connected, seeds, num_nodes_connected: connectedNodesCount } = cluster;
 
-    if (!cluster) {
-      return;
-    }
-
-    return (
-      <div>Remote cluster details</div>
-    );
-  }
-
-  renderCluster() {
-    // const { cluster, intl } = this.props;
+    const renderedSeeds = seeds.map(seed => <EuiText key={seed}>{seed}</EuiText>);
```

Comment:
I made a dumb error in my clusters and added 2 identical seeds, which caused React to complain about the keys here not being unique. It's not necessarily something we need to fix here, but would be good as part of the UI validation to check that the user did not enter duplicate seeds.